### PR TITLE
fix: missing ui features in context menu

### PR DESCRIPTION
**Pull request by sreehari2003**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by dstala** [comment]
cell context menu & view context menu needs a fix

Screenshot 2023-09-15 at 8 28 41 PM
Screenshot 2023-09-15 at 8 30 21 PM

**Comment by mustafapc19** [comment]
@sreehari2003 
For context menu, use `NcMenu` and `NcMenuItem` generic components similar to ProjectNode

**Comment by dstala** [comment]
@sreehari2003 for view context menu, font size is not 0.85rem

**Review by dstala — CHANGES_REQUESTED** [review]
Use generic component for
- Cell right click modal
- Field context menu
- View context menu

**Review by dstala — APPROVED** [review]

File: packages/nc-gui/components/smartsheet/header/Menu.vue [hunk 1]
```diff
@@ -375,7 +375,7 @@ const onInsertAfter = () => {
 
 <style scoped>
 .nc-header-menu-item {
-  @apply text-xs flex items-center px-1 py-2 gap-1;
+  @apply text-dropdown flex items-center px-1 py-2 gap-1;
```
Comment (mustafapc19) [inline packages/nc-gui/components/smartsheet/header/Menu.vue:378]: @sreehari2003 

Remove `text-dropdown` and replace it with `text-sm`

File: packages/nc-gui/windi.config.ts [hunk 2]
```diff
@@ -165,6 +165,9 @@ export default defineConfig({
         dark: colors.dark,
         light: colors.light,
       },
+      fontSize: {
```
Comment (mustafapc19) [inline packages/nc-gui/windi.config.ts:168]: @sreehari2003 Remove this as this is not standard

File: packages/nc-gui/components/smartsheet/grid/Table.vue [hunk 3]
```diff
@@ -1545,7 +1545,7 @@ const expandAndLooseFocus = (row: Row, col: Record<string, any>) => {
                 Clear
               </div>
             </a-menu-item>
-
+            <NcDivider class="!m-0" />
```
Comment (mustafapc19) [inline packages/nc-gui/components/smartsheet/grid/Table.vue:1548]: Remove the margin override and use `NcMenu` and `NcMenuItem` which will take care of this

File: packages/nc-gui/components/smartsheet/grid/Table.vue [hunk 4]
```diff
@@ -1556,14 +1556,16 @@ const expandAndLooseFocus = (row: Row, col: Record<string, any>) => {
...
-
-            <a-menu-item v-else-if="contextMenuTarget && deleteRangeOfRows" @click="deleteSelectedRangeOfRows">
-              <div v-e="['a:row:delete']" class="nc-project-menu-item text-red-600">
-                <GeneralIcon icon="delete" class="text-gray-500 text-error" />
-                <!-- Delete Rows -->
-                Delete rows
-              </div>
-            </a-menu-item>
+            <div v-else-if="contextMenuTarget && deleteRangeOfRows">
+              <NcDivider class="!m-0" />
```
Comment (mustafapc19) [inline packages/nc-gui/components/smartsheet/grid/Table.vue:1560]: Same thing as above

**Review by mustafapc19 — CHANGES_REQUESTED** [review]

**Pull request merged**